sklearn/metrics/scorer.py

```diff
@@ -201,8 +201,7 @@ def _passthrough_scorer(estimator, *args, **kwargs):
     return estimator.score(*args, **kwargs)
 
 
-def check_scoring(estimator, scoring=None, allow_none=False,
```

Comment:
This function is not documented [here](http://scikit-learn.org/dev/modules/classes.html#module-sklearn.metrics)... 

In general could I assume anything that is not documented to not be a part of our public API?